index.html

```diff
@@ -32,13 +32,15 @@
 				<a href="#">My Logo</a>
 			</div>
 			<section class="menu">
-				<nav><img src="Assets/images/Union.png" alt="Menu" class="hamburger">
+				<nav id="Main-menu"><img src="Assets/images/Union.png" alt="Menu" class="hamburger" id="hbg">
 					<!-- responsive web design for desktop screen head section-->
-					<ul class="nav-menu">
-						<li><a href="#">Portfolio</a></li>
-						<li><a href="#">About</a></li>
-						<li><a href="#">Contact</a></li>
+					<ul class="nav-menu" id="drop-down">
```

Comment:
- [ ] On desktop size the hamburger menu button shouldn't be visible.
![Screenshot_20210623-022317_Chrome](https://user-images.githubusercontent.com/58504655/123003191-49360c00-d3cc-11eb-9c0e-5793c8fc1072.jpg)

- [ ] When the user click (or tap) the hamburger button, the mobile menu should appear.
- [ ] When the user click (or tap) the close (X) button, the mobile menu should disappear.
- [ ] When the user click (or tap) any option of the mobile menu, the mobile menu should disappear

Reply:
All this functionality is working perfectly, I see no reason why I would submit a PR without implementing any of the features requested.

Below is a video link to this project of me implementing the features. also, I will try to upload the video here for you. because I don't understand the reason why you would say I didn't implement those functionalities.
https://www.awesomescreenshot.com/video/4227045?key=a88b5e9edf79fc652487cb80fb539639

https://user-images.githubusercontent.com/54026531/123004335-60fd9800-d3ac-11eb-83b0-ac21557ec024.mp4